packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -87,6 +105,13 @@ def pre_fill_label_with_filename_ext(cls, v, values):
 
     class Config:
         extra = Extra.forbid
+        schema_extra = {
+            "example": {
+                "path": "api/0a3b2c56-dbcd-4871-b93b-d454b7883f9f/input.txt",
+                "eTag": "859fda0cb82fc4acb4686510a172d9a9-1",
+                "label": "input.txt",
+            }
+        }
 
 
 class DatCoreFileLink(BaseFileLink):
```

Comment:
question: why no validator to check store is equal to 1 in that case?

Reply:
in case somebody is tempted to construct with other value

Reply:
for DatCoreFileLink?

Reply:
Should do the same, yes ... but I did not dare since i was not using it :-) But now that you mention, I will now.

Reply:
done